repository/kafka/docs/latest/versions.md

```diff
@@ -0,0 +1,5 @@
+### Versions
+
+- Apache Kafka 2.12-2.3.0
+- OpenJDK 8
+- JMX Exporter 0.1.0
```

Comment:
Worth mentioning the KUDO version here too or no? (we mention the SDK version we use in all of our packages)

Reply:
It doesn't use the KUDO as we use SDK in DC/OS.
Different versions of KUDO can deploy the same version of KUDO Kafka

Reply:
but I thought upgrades didn't work until kudo 0.3.3 came out?